monai/transforms/compose.py

```diff
@@ -338,6 +350,7 @@ def __init__(
         overrides: dict | None = None,
         override_keys: Sequence[str] | None = None,
         verbose: bool = False,
```

Comment:
This should be replaced with a logger_name to be consistent with the rest of MONAI